### PR TITLE
[SwiftLint] Fixing Various Violations

### PR DESCRIPTION
**Pull request by adamrushy**
### 🎯 Goal

This is a mixture of violations (whitespace, colon, isEmpty etc...)

### 🧪 Testing

CMD + B

### ☑️ Contributor Checklist

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] This change follows zero ⚠️ policy (required)
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS)

### CODE REVIEW

File: .swiftlint.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ excluded:
   - vendor
   - UISDKdocumentation
   - Tests
+  - Sources/StreamChatUI/Deprecations.swift
```
Comment (bielikb) [inline .swiftlint.yml:13]: Instead of completely excluding this file from swiftlint, lets just disable rules on a file level.

Comment (adamrushy) [reply .swiftlint.yml:13]: Yeah, we can do this. But I was thinking do we even want deprecations to be covered, especially given this file will be removed in the future

Comment (polqf) [reply .swiftlint.yml:13]: +1 on file level

File: DemoApp/DemoUsers.swift [hunk 2]
```diff
@@ -18,7 +18,7 @@ public struct UserCredentials {
 
 public extension UserCredentials {
     static func builtInUsersByID(id: String) -> UserCredentials? {
-        builtInUsers.filter { $0.id == id }.first
+        builtInUsers.first(where: { $0.id == id })
```
Comment (polqf) [inline DemoApp/DemoUsers.swift:21]:
```suggestion
        builtInUsers.first { $0.id == id }
```

File: StreamChatUITestsAppUITests/MockServer/MessageDetails.swift [hunk 3]
```diff
@@ -22,15 +22,15 @@ extension StreamMockServer {
     }
     
     func getMessageDetails(messageId: String) -> [MessageDetail: String] {
-        waitForMessageDetails().filter { $0[.messageId] == messageId }.first!
+        waitForMessageDetails().first(where: { $0[.messageId] == messageId })
```
Comment (polqf) [inline StreamChatUITestsAppUITests/MockServer/MessageDetails.swift:25]:
```suggestion
        waitForMessageDetails().first { $0[.messageId] == messageId }
```

File: StreamChatUITestsAppUITests/MockServer/MessageDetails.swift [hunk 4]
```diff
@@ -22,15 +22,15 @@ extension StreamMockServer {
...
     }
     
     func getMessageDetails() -> [MessageDetail: String] {
         waitForMessageDetails().last!
     }
     
     func removeMessageDetails(messageId: String) {
-        let deletedMessage = messageDetails.filter { $0[.messageId] == messageId }.first!
+        let deletedMessage = messageDetails.first(where: { $0[.messageId] == messageId })
```
Comment (polqf) [inline StreamChatUITestsAppUITests/MockServer/MessageDetails.swift:33]:
```suggestion
        let deletedMessage = messageDetails.first { $0[.messageId] == messageId }
```

File: Sources/StreamChatUI/Gallery/GalleryVC.swift [hunk 5]
```diff
@@ -242,7 +242,7 @@ open class GalleryVC:
     }
     
     override open func viewWillDisappear(_ animated: Bool) {
-        super.viewWillAppear(animated)
+        super.viewWillDisappear(animated)
```
[the remark this answers is not in the document]
Comment (adamrushy) [reply Sources/StreamChatUI/Gallery/GalleryVC.swift:245]: Yeah, SwiftLint caught it @evsaev 👍🏼

File: .swiftlint.yml (hunk 1 above)
Comment (adamrushy) [reply .swiftlint.yml]: Okay, made this change :]

File: StreamChatUITestsAppUITests/MockServer/MessageDetails.swift (hunk 3 above)
Comment (adamrushy) [reply StreamChatUITestsAppUITests/MockServer/MessageDetails.swift:25]: Nice suggestion!

**Review by polqf — APPROVED** [review]

**Review by nuno-vieira — APPROVED** [review]
LGTM! ✅

**Review by bielikb — APPROVED** [review]

**Pull request merged**